utils/models/transformer_models.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2020, Google LLC.
```

Comment:
We may consider create a folder called `stackoverflow_transformer` in https://github.com/google-research/federated/tree/master/fedopt_guide before things stabilize. No strong preference though. @zcharles8 WDYT?